CONTRIBUTING.rst

```diff
@@ -201,4 +201,4 @@ Windows users can find ``make.bat`` in the ``docs`` folder.
 
 You should also have a look at the `Project Jupyter Documentation Guide`__.
 
-__ https://jupyter.readthedocs.io/en/latest/contrib_docs/index.html
+__ https://jupyter.readthedocs.io/en/latest/contributing/content-contributor.html
```

Comment:
@Nishikoh - thank you for this contribution!

I think the intention of this link is to point to the documentation guide, which appears to be here: https://jupyter.readthedocs.io/en/latest/contributing/docs-contributions/index.html

Once updated, I'd be happy to merge.  Thank you.

Reply:
Thank you for the review! Fixed it :D